packages/widgets/src/components/TradingScoreboard/TradingScoreboard.spec.tsx

```diff
@@ -0,0 +1,93 @@
+import { Icon } from '@ssa-ui-kit/core';
+
+import TradingScoreboard from './TradingScoreboard';
+import userEvent from '@testing-library/user-event';
+
+const itemsArr = [
+  {
+    value: 'Binance',
+    title: 'Exchange',
+  },
+  {
+    value: 'Account name',
+    title: 'Account',
+  },
+  {
+    value: 'Grid v7',
+    title: 'Strategy',
+  },
+  {
+    value: 'ETH/USDT',
+    title: 'Pairs',
+  },
+  {
+    value: '25',
+    title: 'Orders',
+  },
+  {
+    value: '27.07.23',
+    unit: '4:22:23 pm',
+    title: 'Start Date',
+  },
+  {
+    value: '30.07.23',
+    unit: '4:22:23 pm',
+    title: 'End date',
+  },
+  {
+    value: '4d 5h 36m 2s',
+    title: 'Launch duration',
+  },
+  {
+    value: '25',
+    unit: '%',
+    title: 'ROI',
+    icon: <Icon name="arrow-up" color="#2CA24D" size={16} />,
+  },
+  {
+    value: '340.025',
+    unit: 'USD',
+    title: 'PNL',
+    icon: <Icon name="arrow-up" color="#2CA24D" size={16} />,
+  },
+];
+
+describe('TradingScoreboard', () => {
```

Comment:
Please add all the tests that we discussed. 
You are missing some scenarios, e.g., when the `renderCard` prop is used.

Reply:
Done

Reply:
done